src/neuroglancer/sliceview/volume/frontend.ts

```diff
@@ -224,4 +225,7 @@ export interface MultiscaleVolumeChunkSource extends MultiscaleSliceViewChunkSou
    * This only makes sense if volumeType === VolumeType.SEGMENTATION.
    */
   getMeshSource: () => MeshSource | null;
+
+  getSkeletonSource?: () => SkeletonSource | null;
```

Comment:
Maybe better to add the function to the other data sources and return `null` there?

Reply:
done